python/python/tests/test_async_client.py

```diff
@@ -9725,7 +9725,7 @@ async def cluster_route_custom_command_slot_route(
         route_class = SlotKeyRoute if is_slot_key else SlotIdRoute
         route_second_arg = "foo" if is_slot_key else 4000
         primary_res = await glide_client.custom_command(
-            ["CLUSTER", "NODES"], route_class(SlotType.PRIMARY, route_second_arg)
+            ["CLUSTER", "NODES"], route_class(SlotType.PRIMARY, route_second_arg)  # type: ignore
```

Comment:
an artifact of merge?

Reply:
Fixes for the failing python CI picked from release branch

Reply:
Do you want to backport https://github.com/valkey-io/valkey-glide/pull/2453 to main as another PR?